queries.sql

```diff
@@ -107,4 +111,24 @@ SELECT * from animals;
 
 Commit;
 
-SELECT * from animals
\ No newline at end of file
+SELECT * from animals
+
+-- ////Picture 4///
+
+SELECT count(*) from animals;
+
+SELECT count(*) from animals
+WHERE escape_attempts = 0;
+
+SELECT avg(weight_kg) from animals;
+
+SELECT neutered, count(*) as escape_attempts
+FROM animals
+WHERE escape_attempts > 0
+GROUP BY (neutered);
+
+SELECT min(weight_kg), max(weight_kg) from animals;
+
```

Comment:
- [ ] Great job so far in your implementation. However, I would like you to take a second look at this query as it doesn't return the correct results. The query is to return the min and max weight of `each` type of animal. In your animals table you have only two types of animals `Pokemon` and `Digimon` and you're to return the min and max weight of each of these types. Consider rewriting your query to return the written results.

    - Also, as I mentioned in my preview comments. For better adherence to the project requirements, it would be more appropriate to follow the instructions and include the `screenshot` within the `pull request summary`. This will ensure that all project-related content is easily accessible and well-organized for reviewers and team members. 📸👀

Happy coding! 🚀😊